Source/CesiumRuntime/Private/Tests/CesiumLoadTest.cpp

```diff
@@ -227,7 +233,9 @@ void createCommonWorldObjects(LoadTestContext& context) {
   context.pawn->AutoPossessPlayer = EAutoReceiveInput::Player0;
 }
 
-bool RunLoadTest(const size_t locationIndex) {
+bool RunLoadTest(
+    std::function<void(LoadTestContext&)> locationSetup,
```

Comment:
I thought that passing in a setup function, rather than an index indicating which setup function to use, was much cleaner and less error prone.